lib/internal/Magento/Framework/Webapi/ServiceInputProcessor.php

```diff
@@ -82,6 +89,7 @@ public function __construct(
         AttributeValueFactory $attributeValueFactory,
         CustomAttributeTypeLocatorInterface $customAttributeTypeLocator,
         MethodsMap $methodsMap,
+        \Magento\Framework\ObjectManager\ConfigInterface $config,
```

Comment:
Hi, @phoenix128 , due to [Magento backward-compatible guide](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/) we can't add required parament to the constructor. 
Please add the optional parameter and take instance from `ObjectManager`, as example:
`$this->storeManager = $storeManager ?: \Magento\Framework\App\ObjectManager::getInstance()>get(\Magento\Store\Model\StoreManagerInterface::class);`